src/common/ModalDialog/styles.less

```diff
@@ -19,6 +19,7 @@
         padding: 0 2rem;
         border-radius: var(--border-radius);
         background-color: var(--modal-background-color);
+        box-shadow: 0px 0px 30px #7B5BF55E;
```

Comment:
I thought about making a variable for this. The reason is we'll have the glow in other places too. Like the NavMenu for example. (--outer-glow: 0px 0px 30px #7B5BF55E) This way it will be way easier to change when we implement themes.